src/V3Clock.cpp

```diff
@@ -151,12 +175,58 @@ class ClockVisitor final : public VNVisitor {
         clearLastSen();
     }
 
+    // Create sampled values
+    virtual void visit(AstNodeModule* nodep) override {
```

Comment:
We don't need to save the module. We can get it from `m_scopep->modp()`.

Reply:
Done.